cmd/rook/ceph/cleanup.go

```diff
@@ -87,3 +111,87 @@ func startCleanUp(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func startSubVolumeGroupsCleanUp(cmd *cobra.Command, args []string) error {
+	rook.SetLogLevel()
+	rook.LogStartupInfo(cleanUpSubVolumeGroupsCmd.Flags())
+
+	ctx := cmd.Context()
+	context := createContext()
+	namespace := os.Getenv(k8sutil.PodNamespaceEnvVar)
+	clusterInfo := client.AdminClusterInfo(ctx, namespace, "")
+
+	fsName := os.Getenv(opcontroller.CephFSVolumeNameEnv)
+	if fsName == "" {
+		return fmt.Errorf("cephfs volume name is not available in the pod environment variables")
+	}
+	subVolumeGroupName := os.Getenv(opcontroller.SubVolumeGroupNameEnv)
+	if subVolumeGroupName == "" {
+		return fmt.Errorf("cephfs SubVolumeGroup name is not available in the pod environment variables")
+	}
+	csiNamespace := os.Getenv(opcontroller.CephFSNamesaceEnv)
+	if csiNamespace == "" {
+		return fmt.Errorf("cephfs SubVolumeGroup namespace name is not available in the pod environment variables")
+	}
+
+	poolName := os.Getenv(opcontroller.CephFSMetaDataPoolNameEnv)
+	if poolName == "" {
+		return fmt.Errorf("cephfs metadata pool name is not available in the pod environment variables")
+	}
+
+	logger.Infof("starting clean up ceph SubVolumeGroup resource %q", subVolumeGroupName)
+
+	subVolumeList, err := cephclient.ListSubvolumesInGroup(context, clusterInfo, fsName, subVolumeGroupName)
```

Comment:
Only the CLI processing should be in the `cmd` package. The rest of this code should likely move under  `pkg/daemon/ceph`

Reply:
moved to `pkg/daemon/ceph`